runtime/autoload/man.vim

```diff
@@ -70,12 +72,14 @@ endfunction
 function! man#read_page(ref) abort
   try
     let [sect, name] = man#extract_sect_and_name_ref(a:ref)
-    let [b:man_sect, name, path] = s:verify_exists(sect, name)
+    let [sect, name, path] = s:verify_exists(sect, name)
     let page = s:get_page(path)
   catch
-    " call to s:error() is unnecessary
+    " call to s:error() does nothing because we were
+    " called from the BufReadCmd autocmd.
```

Comment:
What matters is that an error was already shown. I don't know what changed to make s:error "do nothing" (nor do I see how that's possible, it clearly does something if called). 

The new comment is also 3x as long. What's wrong with the old comment?